master/static/migrations/migration-move-to-top.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I ended up doing something like this a few times to keep up with changes in master. I think it might help to have it in the repo.

Reply:
fine with me